### PR TITLE
Create Hubspot Authlify integration

### PR DESCRIPTION
**Pull request by picocodes**

### CODE REVIEW

File: Hubspot.php [hunk 1]
```diff
@@ -87,11 +87,11 @@ public function getEmailList( $count = 250 )
         $data       = new Data\Collection($response);
         $lists      = $data->filter('lists')->toArray();
 
-        if( empty( $lists ) ) {
+        if(! is_array( $lists ) ) {
             return array();
         }
 
-        $filtered = array();
+        $filtered = [ 'all' => 'All Contancts' ];
```
Comment (collizo4sky) [inline Hubspot.php]: @picocodes   You shouldn’t be adding the all contact entry here. Do that in HubSpot connection in but bucket.

Remove it please

**Comment by picocodes** [comment]
okay

**Comment by picocodes** [comment]
fixed

File: Hubspot.php [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+            throw new InvalidArgumentException('List ID is missing');
+        }
+
+        if (empty($email)) {
+            throw new InvalidArgumentException('Email address is missing');
+        }
+
+        $response = $this->apiRequest("contacts/v1/contact/createOrUpdate/email/$email", 'POST', $contact_data );
+        
+        if( !empty( $response->vid ) && 'all' != $list_id ) {
```
Comment (collizo4sky) [inline Hubspot.php]: @picocodes please do the check for `'all' != $list_id` in the connections lib not here.

Comment (collizo4sky) [reply Hubspot.php]: @picocodes never mind. I fixed it.

**Pull request merged**